packages/observable-md/index.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
```

Comment:
I like the idea of having an `index.html` in a package directory, but this is not standard for the project. Should we aim to have an `index.html` in each package showcasing some simple usage?

Reply:
I see an index.html as being part of docs, so the more the better...